releases/net471/KnownIssues/489937-WPF Touch and Stylus AccessViolation in GetPenEventMultiple.md

```diff
@@ -0,0 +1,12 @@
+## WPF Touch/Stylus Generates AccessViolations and Heap Corruption in GetPenEventMultiple
+
+### Symptoms
+On a touch or stylus enabled machine WPF applications may sometimes corrupt memory and crash in GetPenEventMultiple.
```

Comment:
comma after WPF
Is GetPenEventMultiple public? Can it be a link (`E:namespaces.type.GetPenEventMultiple`)